Dockerfile

```diff
@@ -1,17 +1,20 @@
-ARG CP_KAFKA_CONNECT_BASE_VERSION=6.0.1
+ARG CP_KAFKA_CONNECT_BASE_VERSION=7.2.2.amd64
```

Comment:
**Required**: We **MUST** use the [7.2.2 multi-archi image](https://hub.docker.com/layers/confluentinc/cp-kafka-connect-base/7.2.2/images/sha256-914d0402e233cc0ff5fefbbe62f17c24e17dcb8dfdb19eac75123550ba9c2f15?context=explore) to be able to run on ARM k8s clusters as well, I see no reason to use the amd64 specific one as the cluster should decide on the correctly architectured image based on the platform specified by the cluster at pull time using the multi-arch image.

```suggestion
ARG CP_KAFKA_CONNECT_BASE_VERSION=7.2.2
```

Reply:
Okay, I just realized we are probably not building our own image to multi-arch which seems to be the case.

Even though the [multi-arch arm support would be relatively easy](https://github.com/banzaicloud/logging-operator/pull/657/files) I do understand that would be a feature implementaiton, so consider this request void.